DESCRIPTION

```diff
@@ -1,25 +1,24 @@
 Package: watina
 Title: Querying & Processing Data From The INBO Watina Database (mainly groundwater data)
 Version: 0.0.0.9000
-Authors@R: 
-    person(given = "First",
-           family = "Last",
-           role = c("aut", "cre"),
-           email = "first.last@example.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
 Description: The R-package watina contains functions to query
     and process data from the Watina database at the Research Institute for
     Nature and Forest (INBO). This database provides
     groundwater level and chemical data, mainly from natural 
     areas in Flanders (Belgium).
+Authors@R: c(
+    person("Floris", "Vanderhaeghe", role = c("aut", "cre"), email = "floris.vanderhaeghe@inbo.be", comment = c(ORCID = "0000-0002-6378-6229")),
+    person("Research Institute for Nature and Forest (INBO)", role = "cph", email = "info@inbo.be")
```

Comment:
Right, I still forgot INBO; thanks!
I don't know where I've got it from (probably another INBO package), but in the n2khab package I use the role `c("cph", "fnd")` for INBO (funder). What is your view on this?

Reply:
I would keep it to copyright holder. Note that the contributing guide says we are not directly funded to maintain this pkg. :smile:

Reply:
OK, I'll correct it over there.